### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by SeungheonOh**

### CODE REVIEW

File: Plutarch/Builtin/Data.hs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  = PCons (Term s a) (Term s (PBuiltinList a))
+  | PNil
+
+-- -- instance
+-- --   ( PShow a
+-- --   , PLC.Contains PLC.DefaultUni (PlutusRepr a)
+-- --   ) =>
+-- --   PShow (PBuiltinList a)
+-- --   where
+-- --   pshow' _ x = pshowList @PBuiltinList @a # x
```
Comment (t4ccer) [inline Plutarch/Builtin/Data.hs:84]: To remove

File: Plutarch/ByteString.hs [hunk 2]
```diff
@@ -1,401 +1,278 @@
-{-# LANGUAGE FlexibleInstances #-}
```
Comment (t4ccer) [inline Plutarch/ByteString.hs:1]: Probably whole file can be removed?

**Comment by SeungheonOh** [comment]
Just merge #767 instead of this

**Comment by SeungheonOh** [comment]
superceeded by #767

**Pull request closed without merging**